spec/unit/facter/java_patch_level_spec.rb

```diff
@@ -17,8 +17,8 @@
       end
     end
 
-    context "if java is installed" do
-      context 'returns java patch version extracted from java_version fact' do
+    context "if java is not installed" do
+      context 'returns nil because of non existent java_version fact' do
```

Comment:
look at the whole file ;)
While I was looking at the spec tests, I found this seemed to me to be a cut'n pasto error, that doesn't really reflect what the test is doing, nor what it expects to get back.

I can revert that if you prefer.

cheers